phpunit/phpunit/4.7/phpunit.xml.dist

```diff
@@ -5,10 +5,11 @@
          xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.5/phpunit.xsd"
          backupGlobals="false"
          colors="true"
-         bootstrap="tests/bootstrap.php"
+         bootstrap="bootstrap.php"
 >
     <php>
         <ini name="error_reporting" value="-1" />
+        <env name="APP_ENV" value="test" />
```

Comment:
> APP_ENV is already set in bin/phpunit

APP_ENV is *explicitly* set to "test" in `bin/phpunit` - so that `.env.test` is loaded in both cases.
*Not* adding this line is what would create the inconsistency, by not loading `.env.test` when using `vendor/bin/simple-phpunit` (or `vendor/bin/phpunit` btw if installed.)

Or did I miss your case?

Reply:
OK, fixed by *not* loading .env files in bin/phpunit, so that only phpunit.xml drives them.